Formula/v/virt-manager.rb

```diff
@@ -54,18 +54,25 @@ class VirtManager < Formula
     sha256 "dd505485549a7a552833da5e6063639d0d177c04f23bc3864e41e5dc5f612168"
   end
 
+  def pythons
+    deps.map(&:to_formula)
+        .select { |f| f.name.match?(/^python@\d\.\d+$/) }
+        .map { |f| f.opt_libexec/"bin/python" }
+  end
+
   def install
-    python3 = "python3.12"
-    venv = virtualenv_create(libexec, python3)
-    venv.pip_install resources
+    pythons.each do |python|
```

Comment:
same as above

Reply:
Same as above.

Reply:
yeah, that is what I thought. 

relates to https://github.com/Homebrew/homebrew-core/pull/193697